src/Sources.jl

```diff
@@ -133,11 +133,15 @@ function download_source(source::T; verbose::Bool = false, downloads_dir = stora
         src_path = abspath(source.url)
 
         # And if this is a locally-sourced tarball, just verify
-        verify(src_path, source.hash)
+        verify(src_path, source.hash) || error("Verification failed")
```

Comment:
To be honest, I have no idea why we have this branch.  I'm leaving it as it is, but I don't know when we have sources that are local files

Reply:
Unsurprisingly, these are 2 of the only 3 lines missing in code coverage in this file :smile: